def/core.js

```diff
@@ -360,12 +360,12 @@ def("Comment")
 
 // Block comment. The .type really should be BlockComment rather than
 // Block, but that's what we're stuck with for now.
-def("Block")
+def("CommentBlock")
     .bases("Comment")
     .build("value", /*optional:*/ "leading", "trailing");
```

Comment:
No I mean `Block` and `Line` would move to `def/esprima.js`, since that's where they always probably should have been. The abstract `Comment` supertype can remain in `def/core.js`, though.